### DISCUSSION TITLE
Welcome to the Ansible keycloak Collection Discussions!

### DISCUSSION THREAD

**Discussion by guidograzioli** [ansible-middleware/keycloak#198] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ArnoEv** [comment]
Hi,

I've started to use this collection and I came accros some issues.
It looks like the parameters linked to hostname used in tis collection are deprecated.
There is a v2 for the parameters linked to hostname: https://www.keycloak.org/server/all-config#category-hostname_v2

Any plan to update the collection?

Thanks

**Reply by guidograzioli** [reply]
Hello, yes the plan is to keep the collection up to date with the upstream project. 
Regarding hostname v2, you need to enable the feature hostname:v2, or hostname v1 will be used.
Anyway you are welcome to create an enhancement request here: 
https://github.com/ansible-middleware/keycloak/issues/new?assignees=&labels=&projects=&template=feature_request.md

**Reply by ArnoEv** [reply]
Thanks for your quick reply.

Indeed you are right I could enable the feature, my point was that the parameters in feature hostname:v2 are a bit different and if I enable it I won't be able to use all of them.
For instance in the configuration template:
hostname-strict-backchannel={{ keycloak_quarkus_hostname_strict_backchannel (this parameter is different in hostnamev:2).

I will stay with the default feature for now and see if I create a enhancement request.

Thanks anyway for the collection, very usefull
